package.json

```diff
@@ -224,6 +224,11 @@
                         "200ms"
                     ],
                     "description": "Debounce time for synchronizing mouse selection. Set to 0 to disable synchronization"
+                },
+                "vscode-neovim.statusLineSeparator": {
+                    "type": "string",
+                    "default": " ",
```

Comment:
Default should be `|`

Reply:
Yeah, we may as well pick a good default

Reply:
@xiyaowong The problem is that a big focus of this extension is to reduce the lines of code and simplify, because currently it is difficult to maintain. This means thinking critically of features that don't bring much value. In this case, having an option provides small value. 

If it would be better to simply have spaces for everyone, that would be fine. 

We can merge it, but we should avoid adding configuration options that make little impact.

Reply:
Many users have different preferences for UI, such as me. The maintenance difficulty is not directly related to the new features and the number of lines of code. I agree that our primary goal is to reduce the code complexity as much as possible, but it is by no means relying on reducing the code and avoiding new configuration items.